bedrock/foundation/templates/foundation/base-resp.html

```diff
@@ -44,8 +54,32 @@
         </li>
       {% endfor %}
       </ul>
+      <ul class="nav">
```

Comment:
Sure, this needn't be a blocker, it was mostly a suggestion. These original pages were built a long time ago and reviewing this update has showed me all kinds of things I wish we'd done differently in the first place ;)